AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/actions.js

```diff
@@ -351,6 +351,15 @@ define([
          */
         getLicenseButtonTitle: function () {
             return this.isDownloaded() ?  $.mage.__('License') : $.mage.__('License and Save');
+        },
+
+        /**
+         * Returns image name of previously saved preview image
+         *
+         * @returns {String}
+         */
+        getImageName: function () {
+            return this.preview().displayedRecord()['path'].split('/').pop();
```

Comment:
This logic is based on an assumption and precaution that images can be nested in folders, so will have multiple directory slashes in the path. I can modify it to `return this.preview().displayedRecord()['path'].substring(1);` to remove the leading slash if necessary.

Reply:
@drpayyne Images really can be nested in folders. Image are saved to Media Gallery folder from which Adobe Stock Integration panel is opened.